tools/EMLassemblyline/taxo_cov_template.xml

```diff
@@ -1,8 +1,7 @@
 <tool id="taxo_cov_template" name="Make taxonomic coverage template" version="0.1.0+galaxy0" profile="22.05">
```

Comment:
please make sure you increase all the tool versions - the suffix here to from 0 to 1

Reply:
Done. Thank you!